### PR TITLE
adding correct blocking logic for ffw

### PR DESCRIPTION
**Pull request by th3fallen**
will resolve https://github.com/owncloud/firewall/issues/109

### CODE REVIEW

File: apps/files/js/filelist.js [hunk 1]
```diff
@@ -1123,6 +1123,14 @@
 				return false;
 			}
 
+			// Firewall Blocked request?
+			if (result.status === 403) {
+				// Go home
+				this.changeDirectory('/');
+				OC.Notification.show('Forbidden');
```
Comment (DeepDiver1975) [inline apps/files/js/filelist.js]: l10n missing
proper sentence missing :speak_no_evil:

Comment (th3fallen) [inline apps/files/js/filelist.js]: What would you propose this sentence be?

Also any example of l10n in this relm?

Comment (MorrisJobke) [inline apps/files/js/filelist.js]: Something like ` This operation is forbidden` ... be creative :P

**Comment by th3fallen** [comment]
@DeepDiver1975 @MorrisJobke string changed, and l10n'd

**Comment by cmonteroluque** [comment]
@PVince81 can you please review this PR? This is for a customer

**Comment by PVince81** [comment]
Code looks good :+1:

**Comment by PVince81** [comment]
@nickvergessen @schiesbn @icewind1991 second review ?

**Comment by icewind1991** [comment]
:+1: makes sense

**Pull request merged**